tests/automated_tests/import_example_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+exit_if_not_successful() {
+   if [ $? -ne 0 ]
+   then
+      exit $?
+   fi
+}
+
+./.vscode/scripts/import-example-app.sh seat-adjuster
```

Comment:
don't we need also some tests here? currently you just check if the script is executing without errors. What about checking the files?

Reply:
Actually it is not needed, because If you open the script you will see that script just coping it locally, what is checked by exit code.

Reply:
What Markus means is that you should also verify that the expected files are there after being copied.

Reply:
If cp commands returns 0, it means that is copied, otherwise, it will return non-zero exit code, am I right?

Reply:
yes you should not test if the copy is successful, but test if the correct files are there (AppManifest, Dockerfile and so on)